src/interpreter/Opcode.cpp

```diff
@@ -17,16 +17,19 @@
 #include "Walrus.h"
 
 #include "interpreter/Opcode.h"
+#include "interpreter/ByteCode.h"
```

Comment:
Do you need this?

Reply:
Yes, it is needed beacuse otherwise the `sizeof` of the classes would fail in `opcode.def`.